.circleci/config.yml

```diff
@@ -10,18 +10,163 @@ commands:
       - run:
           name: Combine package-lock.json files to single file
           command: ls -d -- packages/*/*-lock.json | awk -F packages '{printf "\"packages%s\" ", $2}' | xargs cat > << parameters.filename >>
+  slack-basic:
```

Comment:
Probably better to go in it's own commands.yml, but I'll keep an eye on how you do this so I can do the same in audius-protocol. 

Side note: Perhaps we can have a call soon so you can walk me through how you organized your ymls? I noticed `generate-config` today. Do you think it's worth separating out our config.yml within `audius-protocol` today?

Reply:
Yup I think we can put this in the `.circleci/commands/commands.yml` file. Let's chat today about how this is structured, at a high level it's using circleci's dynamic configuration feature: https://circleci.com/docs/dynamic-config

I would say yes it makes sense to break up the `audius-protocol` config as well. It's pretty big already as one file